lib/js/jsconsole.nim

```diff
@@ -110,5 +110,14 @@ since (1, 5):
     {.line: loc.}:
       {.emit: ["console.assert(", assertion, ", ", msg, ");"].}
 
+  func dir*(console: Console; obj: auto) {.importcpp.}
+    ## https://developer.mozilla.org/en-US/docs/Web/API/Console/dir
+
+  func dirxml*(console: Console; obj: auto) {.importcpp.}
+    ## https://developer.mozilla.org/en-US/docs/Web/API/Console/dirxml
+
+  func timeStamp*(console: Console; label: cstring) {.importcpp.}
+    ## https://developer.mozilla.org/en-US/docs/Web/API/Console/timeStamp
```

Comment:
```suggestion
    ## https://developer.mozilla.org/en-US/docs/Web/API/Console/timeStamp
    ## ..warning:: non-standard
```

Reply:
(please double check the syntax for warning; see https://developer.mozilla.org/en-US/docs/Web/API/Console/timeStamp)